mautic/mautic
# Webhook change segment normalize contact data

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
If segment membership is changed directly by `mautic:segment:rebuild` then payload for segment membership is not full. This PR fixed it



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Create webhook for segment change event and add https://webhook.site/ to hook
3. Change segment for contact directly from contact profile
4. Set segment and add filter for add that contact to segment
5. Rebuild segment
6. Check https://webhook.site/  and you see  the full payload and not full payload  for the same event
7. After PR should work properly
